projects/hslayers/src/assets/locales/cs.json

```
@@ -413,7 +413,7 @@
     },
     "dimensions": "Dimenze",
     "editorButtons": "Editační tlačítka",
-    "enablePhysicalLayerList": "Enable/Disable physical layer list",
+    "enablePhysicalLayerList": "Povolit/zakázat fyzický seznam vrstev",
```

```suggestion
    "enablePhysicalLayerList": "Vypnout/Zapnout fyzický seznam vrstev",
```
I think this wording makes better sense in the given context.